src/index.js

```diff
@@ -1,88 +1,89 @@
+  const [result, setResults] = useState();
+  const [resultColor, setResultColor] = useState("black");
+  const [correctAnswer, setCorrectAnswer] = useState(false);
 
-  // = entryAlt[Math.floor(Math.random() * entryAlt.length)];
-  const getOperator = () => {
-    const mathOps = ["+", "-", "*", "/"];
+  const getRandomNr = useCallback(() => {
+    const randomN1 = Math.floor(Math.random() * 20) + 1;
+    const randomN2 = Math.floor(Math.random() * 20) + 1;
+    const operator = getOperator();
+    setMathOperator(operator);
+    setRandomNrs([randomN1, randomN2]);
+    if (operator === "+") {
+      setResults(randomN1 + randomN2);
+    }
+    if (operator === "-") {
+      setResults(randomN1 - randomN2);
+    }
+  }, []);
 
-    return mathOps[Math.floor(Math.random() * mathOps.length)];
-  };
-
-  const handleResult = () => {
-    let result;
-    // const result = document.getElementById("ResultSquare").value;
-    console.log("result");
-    return result;
-  };
-
-  const handleNext = () => {
-    const NextPushed = () => {
-      setCount(count + 1);
-    };
-    NextPushed();
-    return (
-      <div id="board" className="board-row">
-        <NumberSquares value={getRandomNr()}></NumberSquares>
-        <MathSigns value={getOperator()}></MathSigns>
-        <NumberSquares value={getRandomNr()}></NumberSquares>
-        <MathSigns value="="></MathSigns>
-        {gameResultSquare()}
-      </div>
-    );
-  };
+  useEffect(() => {
+    getRandomNr();
+  }, [getRandomNr]);
 
-  const handleCheck = () => {
-    console.log("CHECK button pushed");
-  };
+  const getInput = useCallback((event) => {
+    setUserInput(event.target.value);
+    setCorrectAnswer(false);
+    setResultColor("black");
+  }, []);
 
-  // renderNrSquare(i) {
-  //   return (
-  //     <NumberSquares
-  //       value={this.getRandomNr()}
-  //     />
-  //   );
-  // }
-  const input = (input) => {
-    //stub - nem tudom mi jon ide
-    console.log(input);
-  };
+  const handleNext = useCallback(() => {
+    setCount(count + 1);
+    getRandomNr();
+    setUserInput("");
+    setCorrectAnswer(false);
+    setResultColor("black");
+    return <div className="game-board"></div>;
+  }, [count, getRandomNr]);
 
-  const gameResultSquare = () => {
-    return <ResultSquare onChange={input} />;
-  };
+  const handleCheck = useCallback(() => {
+    if (userInput === `${result}`) {
+      setCorrectAnswer(true);
+      setResultColor("green");
+    }
+  }, [userInput, result]);
 
   return (
     <div className="game">
       <div className="game-board">
         <div>
           <div id="board" className="board-row">
-            <NumberSquares value={getRandomNr()}></NumberSquares>
-            <MathSigns value={getOperator()}></MathSigns>
-            <NumberSquares value={getRandomNr()}></NumberSquares>
-            <MathSigns value="="></MathSigns>
-            {gameResultSquare()}
+            <NumberSquares value={randomNrs[0]}></NumberSquares>
```

Comment:
you can always ensure that you're getting the smallest number and the largest by using something like: `<NumberSquares value={Math.Min(...randomNrs)}></NumberSquares>` and `<NumberSquares value={Math.Max(...randomNrs)}></NumberSquares>`